frontend/src/book-read/components/shared/review-mutation.jsx

```diff
@@ -0,0 +1,138 @@
+import {
+  //   Box,
+  Paper,
+  Rating,
+  TextField,
+  Stack,
+  Button,
+  Typography,
+  Container,
+  Alert,
+} from "@mui/material";
+import { useState, useEffect } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+const initialFormState = {
+  score: 0,
+  title: "",
+  text: "",
+};
+
+export default function ReviewMutationComponent({
+  onSubmit,
+  //   onDelete,
+  //   headerTitle,
+  mutationTitle,
+  error,
+  review,
+  //   isLoading,
```

Comment:
In fact, you can use isLoading for blocking submit button. It is not really important, but it is the only easy-lazy responsivness this app provides.

Reply:
Used isLoading for better UX of the button